src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ CHANGELOG
  * Implementing the `BundleInterface` without implementing the `getPublicDir()` method is deprecated.
    This method will be added to the interface in 5.0.
  * The `DebugHandlersListener` class has been marked as `final`
+ * Getting the container from a non-booted kernel is deprecated.
```

Comment:
we need entries in `UPGRADE-4.4.md` and `UPGRADE-5.5.md` too

Reply:
updated